build.sh

```diff
@@ -206,20 +208,30 @@ if [[ ${#_CXX} > 0 ]]; then
     CC_PREFIX="-DCMAKE_CXX_COMPILER=$_CXX -DCMAKE_C_COMPILER=$_CC"
 fi
 
-build_directory="$CHAKRACORE_DIR/BuildLinux/${BUILD_TYPE:0}"
+root_build_directory="$CHAKRACORE_DIR/Build/clang_build/${BUILD_ARCH}_${BUILD_TYPE:0}"
```

Comment:
>:0 [](start = 82, length = 2)

This is substring starting at character 0 isn't it? I think the `:0` is unnecessary.